### PR TITLE
Update paths in hp_search tutorial

### PR DESCRIPTION
**Pull request by nd7141**
# Description
Updated tutorial with hp_search with correct paths

## Type of change

- [ ] Feature
- [ ] Fix
- [ ] Release

## Checklist

- [ ] I have followed the [CONTRIBUTING](https://github.com/criteo/deepr/blob/master/docs/CONTRIBUTING.rst) guidelines.
- [ ] I have updated the [CHANGELOG](https://github.com/criteo/deepr/blob/master/docs/CHANGELOG.rst).

### CODE REVIEW

**Review by guillaumegenthial — COMMENTED** [review]
The tests fail because the notebook runs as part of the unit tests and this notebook is trying to spin up some yarn applications (not available from github action).
That's why there was a `HAS_HADOOP` somewhere.

File: docs/getting_started/advanced.ipynb [hunk 1]
```diff
@@ -108,63 +108,60 @@
...
    "execution_count": 7,
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_metrics = [\n",
-    "    dpr.metrics.StepCounter(name=\"num_steps\"),\n",
-    "    dpr.metrics.DecayMean(tensors=[\"loss\"], decay=0.98)\n",
-    "]\n",
-    "eval_metrics = [dpr.metrics.Mean(tensors=[\"loss\"])]\n",
-    "final_metrics = [dpr.metrics.Mean(tensors=[\"loss\"])]"
```
Comment (guillaumegenthial) [inline docs/getting_started/advanced.ipynb:167]: don't delete

File: docs/getting_started/advanced.ipynb [hunk 2]
```diff
@@ -227,17 +224,17 @@
     "        },\n",
     "        every_n_iter=100,\n",
     "        use_graphite=False,\n",
-    "        use_mlflow=False\n",
+    "        use_mlflow=use_mlflow # change it to False if you don't want to track it in mlflow\n",
```
Comment (guillaumegenthial) [inline docs/getting_started/advanced.ipynb:227]: set `USE_MLFLOW = False` at the beginning of the notebook and set all `use_mlflow=USE_MLFLOW`

**Comment by rom1504** [comment]
reopen if you work on it again ;)

**Pull request closed without merging**